text/0000-non-exhaustive.md

```diff
@@ -417,6 +417,10 @@ Then we the only valid way of matching will be:
 let Config { 0: width, 1: height, .. } = config;
 ```
 
+We can think of this as lowering the visibility of the constructor to
+`pub(crate)` if it is marked as `pub`, then applying the standard structure
+rules.
```

Comment:
This needs to apply to unit structs as well.

Reply:
Will do; one sec.